runtime/common/src/evm/mod.rs

```diff
@@ -107,3 +107,55 @@ impl<T: pallet_aura::Config<AuthorityId = AuraId>> FindAuthor<H160> for FindAuth
 		None
 	}
 }
+
+/// Passthrough router deployed bytecode as of this state https://github.com/centrifuge/liquidity-pools/tree/d3da102e7bf656fd50feeb888e17f423317aeeb3
```

Comment:
Q: Which code changes on solidity side lead to a required update of the bytecode?

Reply:
Just the ones in the file linked above. Should maybe make that clear in the comment.